include/linux/soundwire/sdw.h

```diff
@@ -405,6 +406,7 @@ struct sdw_slave_prop {
  * command
  * @mclk_freq: clock reference passed to SoundWire Master, in Hz.
  * @hw_disabled: if true, the Master is not functional, typically due to pin-mux
+ * @quirks: bitmask identifying deltas from the MIPI specification
```

Comment:
it's more 'bitmask identifying optional behavior beyond the scope of the MIPI specification'